pkgs/by-name/xw/xwayland-run/package.nix

```diff
@@ -0,0 +1,34 @@
+{ lib
+, fetchFromGitLab
+, meson
+, ninja
+, python3
+}:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "xwayland-run";
+  version = "0.0.2";
+
```

Comment:
```suggestion

  outputs = [ "out" "man" ];

```

Reply:
Actually when I built with this the man directory from result/share disappeared.

Reply:
it is now in the man output which you might need to explicitly choose to build.

Reply:
Oh, understood. Done.